src/components/TouchableButton/TouchableButton.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+
+import React from 'react'
+import { TouchableOpacity, } from 'react-native'
+
+import type { Node, } from 'react'
```

Comment:
`import React, { type Node } from 'react';`

Reply:
fixed

Reply:
@Dekim12 please do the same with all imports